gui/src/components/performance/cortx-performance.vue

```diff
@@ -51,18 +43,15 @@
 <script lang="ts">
 import { Component, Vue, Prop } from "vue-property-decorator";
 import CortxLineChart from "../widgets/line-chart.vue";
```

Comment:
This import statement and also the component "line-chart.vue" can be removed if it is not used anymore.

Reply:
Removed.